client_documents_test.go

```diff
@@ -8,7 +8,8 @@ func TestClientDocuments_Get(t *testing.T) {
 	var indexUID = "TestClientDocuments_Get"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
```

Comment:
Is there a way to store a variable:

```go
var basicConfig = Config{
    	Host:   "http://localhost:7700",
		APIKey: "masterKey",
}
```

that we import in every test file instead of hard-code the value for each test? 😇

Reply:
I can create a global client in `main_test` and use it in every single test. What do you think?

Reply:
It would work perfectly if you run all the tests, a subset or a single test